packages/core/tsup.config.ts

```diff
@@ -52,8 +52,6 @@ export default defineConfig({
     'src/workflows/constants.ts',
     'src/network/index.ts',
     'src/network/vNext/index.ts',
-    'src/storage/libsql/index.ts',
-    'src/vector/libsql/index.ts',
```

Comment:
what's up with this change?

Reply:
I just noticed that we don't actually have these packages in core anymore, this is stale code, so this can be removed